warning and favicon

whydoesthecodenotwork opened the pull request.

whydoesthecodenotwork committed:
warning and favicon

whydoesthecodenotwork committed:
use more id + remove log

probably better for performance idk

whydoesthecodenotwork commented:
note: this uses question.id which was [just added](https://github.com/sitechtimes/course-selection-backend/commit/28143ee2886800bc63a2f6a23849f01c4c49faa9) to backend and does not exist on the azure yet
warning system will not work without question.id

bridgetf7 commented:
> note: this uses question.id which was [just added](https://github.com/sitechtimes/course-selection-backend/commit/28143ee2886800bc63a2f6a23849f01c4c49faa9) to backend and does not exist on the azure yet warning system will not work without question.id

thank you for letting me know

bridgetf7 committed:
Merge branch 'dev' into 248-routerwatch-functions-comparing-survey-responses

bridgetf7 committed:
Revert "Merge branch 'dev' into 248-routerwatch-functions-comparing-survey-responses"

This reverts commit b12423264d46db7a68b4323ba6749fa25d23d8c7, reversing
changes made to 339709e62f1b9d9c9c9f11fefb832989d0fb9f31.

whydoesthecodenotwork committed:
unsaved warning fixed

+ currentAnsweredSurvey is now updated on save
+ navbar fix (save button could disappear before innerhtml was set)
+ loading is now used when fetching survey (why was it not before???)

bridgetf7 reviewed (changes requested):
Could you please ensure that the `missing answers` function works? The `Save answers before exiting ` alert box now works but the `missing answers` errors do not work upon submitting

whydoesthecodenotwork committed:
submit button won't submit when missing
